src/OSPSuite.SimModelComp/OSPSuite_SimModelComp.xml

```diff
@@ -21,6 +21,14 @@
 						<FieldDef Name="ExecutionTimeLimit" DataType="Double" Description="Execution time limit for single simulation run (0= no limit)" DefaultValue="0"/>
 						<Values>0</Values>
 					</Variable>
+					<Variable Key="ValidateWithXMLSchema">
```

Comment:
Configuration file for Matlab/R-Toolbox extend with possibilities to
* enable/disable validation with SimModel-Schema
* enable/disable identifying used parameters (if false: ALL parameters will be marked as used)